liberty-ls/src/main/java/io/openliberty/tools/langserver/resources/BootstrapPropertiesOptions.java

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+* Copyright (c) 2022 IBM Corporation and others.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0
+*******************************************************************************/
+package io.openliberty.tools.langserver.resources;
+
+import java.util.ListResourceBundle;
+import java.util.ResourceBundle;
+
+public class BootstrapPropertiesOptions extends ListResourceBundle {
+ public static ResourceBundle serverenvProps = ResourceBundle.getBundle("io.openliberty.tools.langserver.resources.ServerEnvOptions");
+    
+ public Object[][] getContents() {
+ return resources;
+    }
+
+ private final static Object[][] resources = {
+        { "command.port", "Custom property. Overrides the default behavior of the Liberty server command port." },
```

Comment:
I  am pretty sure that the properties have to be in a `*.properties` file for translation. Those get shipped to translation centers and they return `*_<locale>.properties` files for each supported language.